src/app/app-launcher/services/app-launcher-gitprovider.service.ts

```diff
@@ -0,0 +1,143 @@
+import { Injectable } from '@angular/core';
+import { Http } from '@angular/http';
+import { Observable } from 'rxjs';
+
+import { GitProviderService } from 'ngx-forge';
+
+import {
+  GitHubRepo,
+  GitHubRepoCommit,
+  GitHubRepoDetails,
+  GitHubRepoLastCommit,
+  GitHubRepoLicense,
+  GitHubUser
+} from '../../space/create/codebases/services/github';
+
+// Enable Access-Conrtol-Expose-Headers for CORS to test
+@Injectable()
+export class AppLauncherGitproviderService implements GitProviderService {
+  private static readonly HEADERS = new Headers({
```

Comment:
The backend already does something like this, I don't think we need this functionality in the front-end

Reply:
No, this was just taken from the demo. Ultimately, the providers will be written for OSIO.

The main reason this PR was created is so Corinne can help make this an experimental feature. Updates to the providers will follow.